controllers/users.js

```diff
@@ -276,18 +277,78 @@ const updateSelf = async (req, res) => {
  * @param res {Object} - Express response object
  */
 const postUserPicture = async (req, res) => {
+  const { file } = req;
+  const { id: userId, discordId } = req.userData;
+  const { coordinates } = req.body;
```

Comment:
is there a check in the validator function that verifies the `coordinates` are present in the req body ?

Reply:
This piece of code pre-existed my code changes. Its just I have modified the try-catch thus it is being shown as a new change.